howto/monitor/cos.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud uses [Canonical Observability Stack(COS)](https://charmhub.io/topics/canonical-observability-stack) to enable users to collect and monitor their workloads.
+
+Since Anbox Cloud uses Juju for deployment, by supporting [COS Lite](https://charmhub.io/topics/canonical-observability-stack/editions/lite) integration, we can export observability information with Prometheus, Loki and Grafana.
+
+Currently, only the [AMS charm](https://charmhub.io/ams) supports COS Lite integration. To gather observability metrics for the AMS charm,
+
+1. Deploy the COS Lite bundle. See the [COS Lite tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s) for instructions.
+2. Deploy the AMS charm as described in the [Instrumenting machine charms](https://charmhub.io/topics/canonical-observability-stack/tutorials/instrumenting-machine-charms) tutorial.
```

Comment:
That tutorial is quite confusing as it mixes many things (extending a charm with observability, deploying it and integrating with the cos stack). I am not sure if that serves the need we have here well enough to show how the charm can be related to the COS stack. Wdyt?

Reply:
I still think majority of that tutorial is applicable as the customer needs to know about the grafana agent charm to deploy it first and then relate to the anbox charms. The only thing that is not of their use it point 2 which actually mentions changes to charm code which they are not expected to do. But apart from that everything still stands. So we can copy the information from there make it specfic to anbox and remove the information asking them to make changes to charm ?